lua/lspconfig/julials.lua

```diff
@@ -7,9 +7,15 @@ local cmd = {
   '--history-file=no',
   '-e',
   [[
-    using Pkg
-    Pkg.instantiate()
+    # Load LanguageServer.jl: attempt to load from ~/.julia/environments/nvim-lspconfig
+    # with the regular load path as a fallback
+    ls_install_path = joinpath(
+        get(DEPOT_PATH, 1, joinpath(homedir(), ".julia")),
+        "environments", "nvim-lspconfig"
+    )
+    pushfirst!(LOAD_PATH, ls_install_path)
     using LanguageServer
+    popfirst!(LOAD_PATH)
```

Comment:
Why do we still need all of this if we are passing `--project` to the julia process?

Reply:
We are not passing `--project` actually. It is a bit tricky, because that would interfere with the project discovery on the lines after. I think the only options are the PR as is, or to juggle with `Base.ACTIVE_PROJECT[]` instead in an equally "hacky" way.

Reply:
Sorry, I meant to imply "why shouldn't we just pass project?" 

I think I understand now, Base.active_project() would then return the project.toml containing directory for LanguageServer.jl vs the one containing the project meant to be analyzed/passed to the LanguageServer.jl instance.

For now, this is good. I still think there's a little bit of duplication in functionality between our root detection in lspconfig and the project.toml detection for figuring out the appropriate julia environment.